### PR TITLE
Improve PF4 axis (x & y)

### PR DESCRIPTION
**Pull request by jotak**
- Add unit to Y axis (issue #18)
- Fix space between ticks (issue #25)
- Moved formatting functions to common
- Fix labels style (smaller) to avoid them being overlapped
- Use VictoryPortal to draw labels at higher "zindex"
- Add units to tooltip, smaller font in tooltip too

Imgur

### CODE REVIEW

File: web/common/utils/formatter.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        return formatDataIEC(d3Format, val, 'B');
+      case 'bitrate':
+      case 'bitrate-si':
+        return formatDataSI(d3Format, val, 'bit/s');
+      case 'bitrate-iec':
+        return formatDataIEC(d3Format, val, 'bit/s');
+      default:
+        // Fallback to default SI scaler:
+        return formatDataSI(d3Format, val, unit);
+    }
```
Comment (aljesusg) [inline web/common/utils/formatter.ts]: Can we remove bytes and bitrate cases or are you going to implement then later ?? In both cases this go to default.

Comment (jotak) [reply web/common/utils/formatter.ts]: No it's intentional and it doesn't go to default, it continues on the following case (cf mangoes and papayas here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch  )

Comment (aljesusg) [reply web/common/utils/formatter.ts]: oh you are right, there is not a break or continue; I didn't see it sorry

**Comment by jotak** [comment]
:-)
Thanks @israel-hdez

**Pull request merged**